api/users/serializers.py

```diff
@@ -56,9 +60,15 @@ class Meta:
 class UserListSerializer(serializers.ModelSerializer):
     role = serializers.SerializerMethodField(read_only=True)
     join_date = serializers.SerializerMethodField(read_only=True)
+    if settings.IS_RBAC_INSTALLED:
+        roles = UserRoleSerializer(many=True, read_only=True, source="get_user_roles")
```

Comment:
I don't really like the if-statement that leads to roles not being set on the serializer. Is this how we do it throughout the codebase? I feel like defaulting `roles` to `None` would look better.

Reply:
I don't think this is possible since UserRoleSerializer lives in RBAC.

Reply:
Ok makes sense.